Tests/conf.json

```diff
@@ -2543,29 +2543,34 @@
             "playbookID": "Active Directory Test",
             "integrations": "Active Directory Query v2",
             "instance_names": "active_directory_ninja",
+            "memory_threshold": 100,
```

Comment:
@anas-yousef  - please explain those changes.

Reply:
The build was failing, as can be seen [here](https://code.pan.run/xsoar/content/-/jobs/31474955#L2562) (line 2562), due to insufficient memory.